common/engine/keyboardprocessor/tests/unit/kmx/kmx.cpp

```diff
@@ -362,7 +362,8 @@ std::u16string parse_source_string(std::string const & s) {
         size_t n;
         std::string s1 = s.substr(p - s.begin(), 8);
         v = std::stoul(s1, &n, 16);
-        assert(v >= 0x20 && v <= 0x10FFFF);
+        // Allow deadkey characters onward
+        assert(v >= CODE_ANY && v <= 0x10FFFF);
```

Comment:
```suggestion
        assert(v >= 0x0001 && v <= 0x10FFFF);
```

Don't use `CODE_ANY` because that's a little misleading -- it happens to have the same value but is not what we are actually testing!